packages/pangea-node-sdk/src/services/authn/user/index.ts

```diff
@@ -23,18 +23,22 @@ export default class User extends BaseService {
    * @summary Delete User
    * @description Delete a user.
    * @operationId authn_post_v2_user_delete
-   * @param {AuthN.User.Delete.EmailRequest | AuthN.User.Delete.IDRequest} request - Supported options:
-   *   - email (string): An email address
-   *   - id (string): The identity of a user or a service
-   * @returns {Promise<PangeaResponse<{}>>} - A promise
-   * representing an async call to the endpoint. Contains an empty object.
+   * @param request Supported options:
+   *   - email (string): An email address.
+   *   - id (string): The identity of a user or a service.
+   *   - username (string): A username.
+   * @returns A promise representing an async call to the endpoint. Contains an
+   * empty object.
    * @example
    * await authn.user.delete({
    *   id: "pui_xpkhwpnz2cmegsws737xbsqnmnuwtbm5",
    * });
    */
   delete(
-    request: AuthN.User.Delete.EmailRequest | AuthN.User.Delete.IDRequest
+    request:
```

Comment:
Should do the same that with `request: AuthN.User.Authenticators.Delete.Request`?

Reply:
Looks like we won't be doing that.